cypress/support/pages/learning_store/pageLearningStore.js

```diff
@@ -40,7 +40,23 @@ class Learning_Store {
 	}
 
 	openLearningStoreContent() {
+		cy.get(Learning_Store.#learningStoreContentDetailToContentLink)
+			.should("exist")
+			.then(($el) => {
+				const clickSpy = cy.spy().as("clickSpy");
+
+				$el.on("click", (event) => {
+					event.preventDefault();
+					clickSpy();
+				});
+			});
+		cy.log("Learning store content was clicked.");
+
 		cy.get(Learning_Store.#learningStoreContentDetailToContentLink).click();
+
+		cy.get("@clickSpy").should("have.been.called");
+
```

Comment:
I have no problem with this empty lines, but I think once in a PR of mine it was requested to remove them.
Should they stioll be removed?

Reply:
done